blimflix/client/app/core/sidebar.scss

```diff
@@ -6,7 +6,7 @@
   top: 50px;
```

Comment:
@Migueld1324 please do not include unrelated code into this PR, it should only contain code from watchlist.